PubNub/Network/PNNetwork.m

```diff
@@ -617,6 +639,7 @@ - (void)appendRequiredParametersTo:(PNRequestParameters *)parameters {
     
     [parameters addPathComponents:self.defaultPathComponents];
     [parameters addQueryParameters:self.defaultQueryComponents];
+    [parameters addQueryParameters:[self.client.telemetryManager operationsLatencyForRequest]];
```

Comment:
Ah, I totally missed this part. Now the rest of this makes more sense. Leaving this here for me.